app/app/settings.py

```diff
@@ -189,37 +190,63 @@
 
 LANGUAGES = [('en', _('English'))]
 
-if not ENV in ['local', 'test']:
+AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default='')
+AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY', default='')
+AWS_DEFAULT_REGION = env('AWS_DEFAULT_REGION', default='us-west-2')
+AWS_LOG_GROUP = env('AWS_LOG_GROUP', default='Gitcoin')
+AWS_LOG_LEVEL = env('AWS_LOG_LEVEL', default='DEBUG')
+AWS_LOG_STREAM = env('AWS_LOG_STREAM', default=f'{ENV}-web')
+
+if ENV not in ['local', 'test']:
+    boto3_session = Session(
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+        region_name=AWS_DEFAULT_REGION
+    )
+
     LOGGING = {
         'version': 1,
         'disable_existing_loggers': False,
+        'root': {
+            'level': 'ERROR',
```

Comment:
So we are not logging the warnings? 😅

Reply:
This is the root/default logging behavior and aligns with the default logger provided by Django as far as I know. 🤔

Reply:
Watchtower is set to log whatever the level is that's passed to: `AWS_LOG_LEVEL` in this case: `DEBUG` - So, much more verbose than what we're even logging currently. @SaptakS

Reply:
Okay. Makes sense.